documentation/book/con-cluster-operator-rbac.adoc

```diff
@@ -52,7 +52,7 @@ include::examples/install/cluster-operator/050-Deployment-strimzi-cluster-operat
       # ...
 ----
 
-Note line 12, where the the `strimzi-cluster-operator` `ServiceAccount` is specified as the `serviceAccountName`.
+NOTE: line 12, where the the `strimzi-cluster-operator` `ServiceAccount` is specified as the `serviceAccountName`.
```

Comment:
I don't think you meant to include this change.

Reply:
Right! thanks